Microsoft.Dynamics365.UIAutomation.Api.UCI/Elements/Entity.cs

```diff
@@ -523,9 +523,9 @@ public void SelectForm(string formName)
         /// </summary>
         /// <param name="controls">The activityparty lookup field name, value or index of the lookup.</param>
         /// <example>xrmApp.Entity.AddValues(new LookupItem[] { new LookupItem { Name = "to", Value = "A. Datum Corporation (sample)" } });</example>
-        public void AddValues(LookupItem[] controls, int index = 0)
+        public void AddValues(LookupItem[] controls)
         {
-            _client.AddValues(controls, index);
+            _client.AddValues(controls);
```

Comment:
Having a extra parameter index make confusing what index to use. I keep just the index inside the control